ee/api/billing.py

```diff
@@ -131,13 +131,52 @@ def deactivate(self, request: Request, *args: Any, **kwargs: Any) -> HttpRespons
             if len(e.args) > 2:
                 detail_object = e.args[2]
                 return Response(
-                    {"statusText": e.args[0], "detail": detail_object.get("error_message", detail_object)},
+                    {
+                        "statusText": e.args[0],
+                        "detail": detail_object.get("error_message", detail_object),
+                        "code": detail_object.get("code"),
+                    },
                     status=status.HTTP_400_BAD_REQUEST,
                 )
             else:
                 raise e
         return self.list(request, *args, **kwargs)
 
+    @action(methods=["GET"], detail=False)
+    def get_open_invoices(self, request: Request, *args: Any, **kwargs: Any) -> HttpResponse:
```

Comment:
Should this also be `get_invoices` and pass the status through?

Reply:
Yeah, that's a good point.